common/djangoapps/pipeline_mako/templates/static_content.html

```diff
@@ -96,6 +96,7 @@
         try:
             return Template("""
                 {% load render_bundle from webpack_loader %}
+                {% render_bundle "commons" %}
```

Comment:
@andy-armstrong: I'm not sure (I'm really new to Webpack in general). Could you please point me to an example of when this happens in our app so I can test it?

Thank you!